libs/service-portal/document-provider/src/lib/messages.ts

```diff
@@ -7,27 +7,27 @@ export const m = defineMessages({
     defaultMessage: 'Skjalaveita',
   },
   documentProviders: {
-    id: 'service.portal:document-provider-document-providers',
+    id: 'sp.document-provider:document-provider-document-providers',
     defaultMessage: 'Skjalaveitur',
   },
   documentProviderSingle: {
-    id: 'service.portal:document-provider-document-provider-single',
+    id: 'sp.document-provider:document-provider-document-provider-single',
```

Comment:
Since you're changing those ids, doesn't it make sense to remove the duplications?
```suggestion
    id: 'sp.document-provider:document-provider-single',
```